facebookincubator/create-react-app
# Add feature #3116 extended Jest config

To verify this change:
1) I ran an e2e test using local copy of create react app and verified the test case breaks by adding `resetMocks` & `resetModules` to `package.json`
2) Made my changes, then I created a new app using the update version of create react app and re-run the test case by adding `resetMocks` & `resetModules` to `package.json` to verify they are now accepted. 
3) Added a an unsupported option such as `setupFiles` and verified user still gets error message.
